### PR TITLE
Remove MouseControl and use OrbitControls instead

### PR DESCRIPTION
**Pull request by yushijinhun**

### CODE REVIEW

File: types/orbit_controls.d.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+	public autoRotate: boolean;
+	public autoRotateSpeed: number;
+
+	public enableKeys: boolean;
+	public keys: { LEFT: number, UP: number, RIGHT: number, BOTTOM: number };
+
+	public mouseButtons = { ORBIT: THREE.MOUSE, ZOOM: THREE.MOUSE, PAN: THREE.MOUSE };
+
+	constructor(object: THREE.Camera, domElement: HTMLElement);
```
Comment (g-plane) [inline types/orbit_controls.d.ts]: `domElement` is optional.

File: types/orbit_controls.d.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	public rotateSpeed: number;
+
+	public enablePan: boolean;
+	public keyPanSpeed: number;
+
+	public autoRotate: boolean;
+	public autoRotateSpeed: number;
+
+	public enableKeys: boolean;
+	public keys: { LEFT: number, UP: number, RIGHT: number, BOTTOM: number };
```
Comment (g-plane) [inline types/orbit_controls.d.ts]: Use number constant directly just like `{ LEFT: 37, UP: 38, RIGHT: 39, BOTTOM: 40 }`.

File: types/orbit_controls.d.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+	public mouseButtons = { ORBIT: THREE.MOUSE, ZOOM: THREE.MOUSE, PAN: THREE.MOUSE };
```
Comment (g-plane) [inline types/orbit_controls.d.ts]: Use `{ ORBIT: THREE.MOUSE.LEFT, ZOOM: THREE.MOUSE.MIDDLE, PAN: THREE.MOUSE.RIGHT }`.

File: types/orbit_controls.d.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+	constructor(object: THREE.Camera, domElement: HTMLElement);
+
+	public getPolarAngle(): number;
+	public getAzimuthalAngle(): number;
+
+	public saveState(): void;
+	public reset(): void;
+
+	public update(): void;
```
Comment (g-plane) [inline types/orbit_controls.d.ts]: Return type is `boolean`.

File: types/orbit_controls.d.ts (hunk 3 above)
Comment (yushijinhun) [reply types/orbit_controls.d.ts]: I have changed it to:
```typescript
mouseButtons: { ORBIT: THREE.MOUSE, ZOOM: THREE.MOUSE, PAN: THREE.MOUSE };
```

I think it should just be a type definition, but not tell the user what the default value is.

File: types/orbit_controls.d.ts (hunk 2 above)
Comment (yushijinhun) [reply types/orbit_controls.d.ts]: Same as below, I think it should just be a type definition, but not tell the user what the default value is.

**Pull request merged**